DOCSP-48750  Add GCP details to Network Security page (#162)
mongodb/docs-atlas-architecture: source/high-availability.txt

.. _arch-center-distribute-data:

Distribute Data Across At Least Three Data Centers in Different Availability Zones
``````````````````````````````````````````````````````````````````````````````````
MongoDB provides high availability by storing multiple copies of data across replica sets.
To ensure that members of the same replica set don't share the same resources, and that 
a replica set can elect a primary if a data center becomes unavailable, you must distribute 
nodes across at least three data centers. We recommend that you use at least five data centers.

You can distribute data across multiple data centers by deploying your {+cluster+} to a region that supports multiple availability zones. 
Each availability zone typically contains one or more discrete data centers, each with redundant power, networking and connectivity, often housed in separate
facilities. When you deploy a dedicated {+cluster+} to a region that supports availability zones, {+service+} automatically splits your {+cluster+}'s 
nodes across the availability zones. For example, for a three-node replica set {+cluster+} deployed to a region with three availability zones, 

> {+service+} deploys one node in each zone. This ensures that members of the same replica set are not sharing resources, so that your database remains 

Reviewer: ```suggestion
{+service+} deploys one node in each zone. This ensures that members of the same replica set aren't sharing resources, so that your database remains 
```

available even if one zone experiences an outage.

To understand the risks associated with using fewer than three data centers, consider the following diagram, 
which shows data distributed across two data centers:

.. figure:: /includes/images/two-data-centers.png
   :figwidth: 750px
   :alt: An image showing two data centers: Data Center 1, with a primary and a secondary node, and Data Center 2, with only a secondary node

> In the previous diagram, if Data Center 2 becomes unavailable, a majority of replica set members remain available and {+service+} can elect a primary.

Reviewer: ```suggestion
In the previous diagram, if Data Center 2 becomes unavailable, a majority of the replica set members remain available and {+service+} can elect a primary.
```

MongoDB allows you to specify the level of acknowledgment requested
for write operations by using :manual:`write concern 
</reference/write-concern/>`. For example, if
you have a three-node replica set with a write concern of

> ``majority``, every write operation needs to be persisted on 

Reviewer: ```suggestion
``majority``, every write operation must be persisted on 
```
